src/test/java/fixtures/DecodesHelper.java

```diff
@@ -140,13 +146,53 @@ public static Arguments getScript(String testName) throws DecodesScriptException
             }
         }
         
+        RawMessage rawMessage = prepareRawMessage(input, decodesScript);
+        DecodedMessage decodedMessage = decodesScript.decodeMessage(rawMessage);
+        return arguments(testName,decodesScript,rawMessage,decodedMessage,assertions);
+    }
+
+    private static URL getInputResource(final String path, final String testName)
+    {
+        String[] extensions = {".input_iridium", ".input_goes", ".input_edl", ".input_data-logger", ".input"};
+        for( String ext: extensions) {
+
+            URL input = DecodesScript.class.getResource(path + testName + ext);
+            if( input != null )
+                return input;
+        }
+        return null;
+    }
+
+    private static String getMediumFromResource(final URL input)
+    {
+        String inputStr = input.toString();
+        if( inputStr.contains("input_") ) {
```

Comment:
style issues,
- { not on next line
- missing brackets (though I honestly waffle on that one.)

Reply:
What bracket is missing here?

Reply:
starting on line 173. for line 169 it's end of line instead of newline.